apps/src/applab/applab.js

```diff
@@ -666,6 +674,17 @@ Applab.init = function(config) {
     }
     drawDiv();
 
+    // Ignore the user's levelHtml for levels without design mode. levelHtml
+    // should never be present on such levels, however some levels do
+    // have levelHtml stored due to a previous bug. HTML set by levelbuilder
+    // is stored in startHtml, not levelHtml.
+    if (config.level.hideDesignMode) {
+      config.level.levelHtml = '';
+    }
+
+    // #designModeViz doesn't exist until after drawDiv() is called.
```

Comment:
Oops, this comment is wrong -- designModeViz is created in StudioApp.prototype.configureDom, which is when the entire page gets rendered, so designModeViz can't really get created any earlier. I'll update the comment.

I chose this location to call Applab.setLevelHtml because it was a place after configureDom and before reset where we already had a hook back into applab code.